### PR TITLE
Fix issue of epfmt #262

### LINKED ISSUE #262: go-sysl ints --epa option shows name of the endpoint in the diagram
### Steps to reproduce the behaviour
```
System_a:
    endpoint:
        System_b <- endpoint

System_b:
    endpoint:
      System_c <- endpoint

    endpoint_2:
      System_e <- endpoint

System_c:
    endpoint:
      System_d <- endpoint

    endpoint_2: ...

System_d:
    endpoint: ...

System_e:
    endpoint:
      System_c <- endpoint_2

Project [appfmt="%(appname)", epfmt="%(epname)", indirect_arrow_color="silver"]:

    no_passthrough:
        System_a

    passthrough_c [passthrough=["System_c"]]:
        System_a

    passthrough_d [passthrough=["System_d"]]:
        System_a

    passthrough_c_e [passthrough=["System_c", "System_e"]]:
        System_a

```
py-sysl:
`sysl ints -v -epa  -o 'py-%(epname)-epa.png' -j Project d.sysl`

go-sysl:
`ints --epa  -o 'go-%(epname)-epa.png' -j Project d.sysl`

**NOTICE**
`no_passthrough` in the image

### Expected behaviour

py-no_passthrough-epa

### Actual behaviour

go-no_passthrough-epa

### PR DESCRIPTION
**Pull request by JackLeeHal**
Fixes #262

Changes proposed in this pull request:
- fix the wrong behaviour when show epfmt

@anz-bank/sysl-developers

### CODE REVIEW

File: sysl2/sysl/ints_view_test.go [hunk 1]
```diff
@@ -417,7 +417,7 @@ state "test" as X_1 {
   state "test" as _2
 }
 _0 -[#grey]-> _1
-_1 -[#black]> _2 : 
+_1 -[#black]> _2
```
Comment (sahejsingh) [inline sysl2/sysl/ints_view_test.go]: I dont like the fact that there was change in behaviour and only one test was broken.

Can you please add tests related to this specific functionality?

Comment (JackLeeHal) [reply sysl2/sysl/ints_view_test.go]: @sahejsingh In fact, I was just deal with that format in wrong way, the ` : ` should be build within `label`.
But, I'll add a test case when `label` is not empty anyway.

**Comment by JackLeeHal** [comment]
@sahejsingh Review needed.
